apis/python/src/tiledbsoma/_tdb_handles.py

```diff
@@ -139,7 +139,7 @@ def _from_soma_object(
     ) -> Self:
         uri = soma_object.uri
         mode = soma_object.mode
-        timestamp = soma_object.timestamp
+        timestamp = context._open_timestamp_ms(soma_object.timestamp)
```

Comment:
when called from the reopen call path, won't this use the timestamp from the original open, rather than using the "current" timestamp?  (aka time.now())

Apologies if I'm misreading this - still tracing all the code paths :-)